### DISCUSSION TITLE
How can I prevent NameTagged hostile mobs from being killed by other players in claims?

### DISCUSSION THREAD

**Discussion by MCHalfAssed** [GriefPrevention/GriefPrevention#741] [Q&A] [answered]
Other players are able to kill hostile mobs in claims - Even though they are nametagged. Is this intentional? If so, is there a way to protect all nametagged mobs, hostile or not? Thank you.

**Comment by RoboMWM** [comment]
As long as hostile mobs have the ability to damage players, players will remain free to retaliate.

**Comment by RoboMWM** [comment, marked as answer]
My players protect such mobs with fences and/or glass.
